lib/renderer.js

```diff
@@ -198,11 +198,15 @@
      * Either expand to allow all dimensions or have a fixed limit base on a config flag
      *
      * @param {Bounds} bounds
+     * @param {number} scaleX
+     * @param {number} scaleY
      * @return {number}
      */
-    BaseRenderer.prototype._getPixmapMaxDimensions = function (bounds) {
+    BaseRenderer.prototype._getPixmapMaxDimensions = function (bounds, scaleX, scaleY) {
```

Comment:
Since this method returns only a single dimension, can we un-pluralize the name to `_getPixmapMaxDimension()`?